pages.pt_BR/common/pdftk.md

```diff
@@ -0,0 +1,25 @@
+# pdftk
+
+> Conjunto de utilitários para manipular ficheiros PDF.
+> Mas informação em: <https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit>.
+
+
```

Comment:
I don't think we're using 2 blank lines between the main description and the commands, but please correct me if I'm wrong :)